src/checkstatus/get_dbstatus_function.py

```diff
@@ -14,11 +19,15 @@ def lambda_get_dbinstance_status(event, context):
         result["task"] = eval_dbinstance_status(rds, context, taskname, identifier)
         result['identifier'] = identifier
         result['taskname'] = taskname
+        logger.info('## FUNCTION RESULT')
+        logger.info(result)
```

Comment:
can the 2 statements be combined into a single log and follow the pattern of `function <function_name> execution result: <result>`
Example: `function eval_dbinstance_status execution result: <result>`